modules/core/src/minmax.cpp

```diff
@@ -1565,13 +1565,24 @@ void cv::minMaxLoc( InputArray _img, double* minVal, double* maxVal,
 {
     CV_INSTRUMENT_REGION();
 
-    CV_Assert(_img.dims() <= 2);
+    int dims = _img.dims();
+    CV_Assert(dims <= 2);
```

Comment:
`CV_CheckLE(dims, 2, "")` is preferable.

Reply:
Done. Thank you.